plugins/aws/providers.go

```diff
@@ -0,0 +1,143 @@
+package aws
```

Comment:
I'd be more in favor of putting these "provider" util functions at the bottom of `sts_provisioner.go` to make that more of an implementation detail. By having `providers.go` in this package, it looks like we're introducing yet another concept in the shell plugins SDK (plugins, provisioners, importers, providers, etc.), which is not the case.

Reply:
Done, thanks!